### ISSUE TITLE
Chartkick should be a Vue plugin

### ISSUE THREAD

**Issue by nkconnor** [ankane/vue-chartkick#4]
Why isn't the package registered as a regular plugin?

https://vuejs.org/v2/guide/plugins.html

**Comment by ankane** [comment]
Hey @nkconnor, this library provides components, which is different than a plugin from what I understand. If you have an example of a popular library that adds components that's also a plugin, please share.

**Comment by nkconnor** [comment]
If you checkout the `awesome-vue` page a lot of those are using the Plugin API, e.g.:

https://github.com/arvidkahl/vue-json-tree-view
https://github.com/haydenbbickerton/vue-charts

I think the basics for Chartkick would be like:

```
var install = function (Vue, options) {
    createComponent("line-chart", Chartkick.LineChart)
    createComponent("pie-chart", Chartkick.PieChart)
    createComponent("column-chart", Chartkick.ColumnChart)
    ....
}
```

I only noticed because I was having dependency issues with the global Vue import `var Vue = require("vue")`

**Comment by ankane** [comment]
Hey @nkconnor, I've tried converting it to plugin on master. You should be able to do:

```js
var Chartkick = require("chartkick")
var VueChartkick = require("vue-chartkick")

Vue.use(VueChartkick, {Chartkick: Chartkick})
```

Let me know how it goes. If it doesn't work, would love your help with it.

**Comment by MatthewJamesBoyle** [comment]
Hi @ankane, not NkConnor but also trying to get this working with webpack. In my main.js
```
const Chartkick = require('chartkick');
const VueChartkick = require('vue-chartkick');
Vue.use(VueRouter);
Vue.use(VueChartkick, { Chartkick });
```
graph.vue
`<template>
  <pie-chart :data="[['Blueberry', 44], ['Strawberry', 23]]"></pie-chart>
</template>
<script>

</script>
<style>
</style>
`

console error:
`Uncaught TypeError: Vue.component is not a function`

doesn't seem to be quite working unfortunatley :(

**Comment by nkconnor** [comment]
@MatthewJamesBoyle, can you share your project? I will help debug

**Comment by ankane** [comment]
@nkconnor does it work for you now?

**Comment by ankane** [comment]
Tested and released 0.2.0.
